django/contrib/sitemaps/management/commands/ping_google.py

```diff
@@ -7,6 +7,12 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('sitemap_url', nargs='?')
+        parser.add_argument('-d', '--site_domain', default=None)
+        parser.add_argument('--use_http', default=False, action='store_false')
```

Comment:
Options usually use a `-` for separators.
```suggestion
        parser.add_argument('--use-http', default=False, action='store_false')
```